crates/typst/translations/zh.txt

```diff
@@ -4,4 +4,5 @@ equation = 式
 bibliography = 参考文献
 heading = 小节
 outline = 目录
-raw = 代码
\ No newline at end of file
+raw = 代码
+# page = 
```

Comment:
In Chinese we use `第 %d 页` to refer to a page. It roughly means `The nth page` in English. Actually the "heading" supplement text also need to be put *after* the ref number, i.e. `%d 小节` rather than `小节 %d`. So I suggest doing either of the following:

- Change logic to support both "before supplement" and "after supplement" so we can support zh & zh-TW correctly inside translation files. Or
- Set the `heading` and `page` translation to empty string, so ref becomes just number. And let users define the right function, or write the supplement text manually.

Reply:
I think your first suggestion is ideal, and I'll have a look at implementing it. But it isn't in the scope of this PR.

I'm not sure the translations will be parsed properly if I set it to an empty string. Perhaps for now then I leave it as missing (so it will fallback to English) until a future PR solves this problem.

Reply:
> I think your first suggestion is ideal, and I'll have a look at implementing it.

I created #5102 so you can track it here.